src/pykeen/ablation/ablation.py

```diff
@@ -421,7 +429,7 @@ def prepare_ablation(  # noqa:C901
             the paths to the training, testing, and validation data.
     """
     directory = normalize_path(path=directory)
-    if isinstance(datasets, str):
+    if isinstance(datasets, (str, dict)):
```

Comment:
what does this dict do?

Reply:
[15578d3](https://github.com/pykeen/pykeen/pull/1374/commits/15578d373fc602e218ccb4343d363174713ace44)

They are a dictionary of paths for training, validation, and testing. This was implemented but not documented.

Reply:
maybe we should use a typeddict for this to make it more implicitly documented

Reply:
In theory, we already have constants and a type here
https://github.com/pykeen/pykeen/blob/1aa363e1defeae8dd7f6add9a30f9776776e42c7/src/pykeen/typing.py#L113-L117

but I think it would be misleading to re-use the type under this name.

Reply:
170a360...a103223